docs/docs/reference/policy-groups.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Policy Groups
```

Comment:
I'd make this part of the `policies` reference guide. Or at least make sure that somebody reading the policies part don't miss this one.